manifests/service.pp

```diff
@@ -11,10 +11,30 @@
   String $jobs_service = $::foreman::jobs_service,
   Stdlib::Ensure::Service $jobs_service_ensure = $::foreman::jobs_service_ensure,
   Boolean $jobs_service_enable = $::foreman::jobs_service_enable,
+  Boolean $jobs_sidekiq = $::foreman::jobs_sidekiq,
 ) {
+  if $jobs_sidekiq {
+    $sidekiq_ensure = $jobs_service_ensure
+    $sidekiq_enable = $jobs_service_enable
+    $dynflowd_ensure = 'stopped'
+    $dynflowd_enable = false
+  } else {
+    $dynflowd_ensure = $jobs_service_ensure
+    $dynflowd_enable = $jobs_service_enable
+    $sidekiq_enable = false
+    $sidekiq_ensure = 'stopped'
+  }
+  service { 'dynflow-sidekiq@orchestrator':
```

Comment:
I now realize that these services might be a bit more complicated. If we don't install the package, they won't exist. That means these should be within the `if $jobs_sidekiq`. The dynflow service always exists so can be outside of the `if`

Reply:
yeah righ, did that, just unsure how to clean them up... If only we would have an `foreman-maintain cleanup` taks :thinking: :rofl: